eox_tenant/models.py

```diff
@@ -65,6 +67,115 @@ def get_microsite_for_domain(cls, domain):
         return microsites[0] if microsites else None
 
 
+class TenantConfigManager(models.Manager):
+    """
+    Custom managaer for Tenant Config model.
+    """
+
+    def get_configurations(self, domain):
+        """
+        Execute optimized query to get site configurations.
+        """
+        configurations = {}
+
+        with connection.cursor() as cursor:
+            cursor.execute("""
+                SELECT
+                    eox_config.id,
+                    eox_config.external_key,
+                    eox_config.lms_configs,
+                    eox_config.studio_configs,
+                    eox_config.theming_configs,
+                    eox_config.meta
+                FROM eox_tenant_tenantconfig eox_config
+                WHERE eox_config.id = (SELECT eox_tenant_route.config_id from eox_tenant_route
+                WHERE eox_tenant_route.domain=%s)""", [domain])
```

Comment:
is this safe for SQL injection?

Reply:
Yes, it is safe. Check references here:
https://docs.djangoproject.com/en/dev/topics/db/sql/#executing-custom-sql
https://docs.djangoproject.com/en/dev/ref/models/expressions/#django.db.models.expressions.RawSQL